lib/texts.js

```diff
@@ -152,6 +152,8 @@ ${leaves}`,
 	'errorType.syntaxError.wrongBody': template`Body field value is exceeding 4KB limit. ${0}`,
 	'errorType.syntaxError.wrongUrl': template`This does not look like a valid URL. ${0}`,
 	'errorType.syntaxError.modifierMissing': template`${0} chain missing modifier.`,
+	'errorType.suitestCommand': template`provided for 'suitest ${0}' command. It`,
+	'errorType.suitestRestArgs': 'at least one user argument should be provided',
```

Comment:
1. This is not a sentence. Add capital letter and a dot at the end if it should be.
2. Would the user know what a "user argument" is? I suggest improving this message with examples or additional help text.